lib/engine/config/game/g_1889.rb

```diff
@@ -231,7 +231,7 @@ module G1889
       "name": "Mitsubishi Ferry",
       "value": 30,
       "revenue": 5,
-      "desc": "Player owner may place the port tile on a coastal town (B11, G10, I12, or J9) without a tile on it already, outside of the operating rounds of a corporation controlled by another player. The player need not control a corporation or have connectivity to the placed tile from one of their corporations. This does not close the company.",
+      "desc": "Player owner may place the port tile on a coastal town (B11, G10, I12, or J9) without a tile on it already, outside of the operating rounds of a corporation controlled by another player. The player need not control a corporation or have connectivity to the placed tile from one of their corporations. This does not close the corporation.",
```

Comment:
I fixed my description but looks like you missed one `company` to `corporation` so fixing.

Reply:
no this references the company as corporations don't close

Reply:
Yep.... dumb. Fixed.